conjurinc/cli-ruby
# Compact output by default when listing objects

**What is this?**

The new default behavior when listing objects is to display the object ids only.

Full object information can be displayed using `--inspect`.

**How to verify?**

You can create a sandbox using `./dev/start.sh`. Then load a policy, and list objects using `conjur list`. Note that the output is non-verbose.

The API key of the `admin` user in the `cucumber` account can be obtained using the Docker logs of the possum container.
